server/index.ts

```diff
@@ -11,6 +12,27 @@ app.use(express.static(path.join(__dirname, '../dist')));
 
 app.use(morgan('combined'));
 
+const config = {
+  authRequired: false,
+  auth0Logout: true,
+  baseURL: 'http://localhost:3000',
+  clientID: process.env.AUTH_ID,
+  issuerBaseURL: 'https://dev-lgwctxldh5a6hwd4.us.auth0.com',
```

Comment:
Perhaps putting the `baseUrl` and `issuerBaseUrl` into `.env` variables would be beneficial so they are reusable